Material.java

```diff
@@ -0,0 +1,32 @@
+package by.gsu.epamlab;
+
+import java.util.Objects;
+
+public class Material {
+    private String name;
+    private double DENSITY;
+
+    public Material(String name) {
+        this.name = name;
+        if (Objects.equals(name, "steel"))
+            DENSITY = 7850.0;
+        else if (Objects.equals(name, "copper"))
+            DENSITY = 8500.0;
+    }

```

Comment:
Of course, we have all info about all objects in the exercise, but, i think, it would be better to make a standart constructor and initialize materials by hardcoding in Runner